packages/blade-old/src/atoms/TextInput/TextInput.native.js

```diff
@@ -395,7 +375,7 @@ const TextInput = React.forwardRef(
                                   placeholderTextColor={placeholderTextColor}
                                   onFocus={onTextInputFocus}
                                   onBlur={onTextInputBlur}
-                                  onChangeText={onChangeText}
+                                  onChangeText={onChange}
```

Comment:
`onChange` and `onChangeText` are two different functions , one contains nativeEvent properties and shouldn't be mergd together even in blade

Reply:
onchange we are getting here is a prop of blade component which works as onChangeText. (this was used in same way earlier as well).